yarn.lock

```diff
@@ -5,15 +5,13 @@
 "@apollo/react-common@^3.1.3":
   version "3.1.3"
   resolved "https://registry.yarnpkg.com/@apollo/react-common/-/react-common-3.1.3.tgz#ddc34f6403f55d47c0da147fd4756dfd7c73dac5"
-  integrity sha512-Q7ZjDOeqjJf/AOGxUMdGxKF+JVClRXrYBGVq+SuVFqANRpd68MxtVV2OjCWavsFAN0eqYnRqRUrl7vtUCiJqeg==
```

Comment:
this is strange... can i ask which version of yarn are you using?

Reply:
I have rebuilt this with the latest, I had auto env resolving to a much older release (1.9.4).